keyboards/crkbd/keymaps/ardakilic/keymap.c

```diff
@@ -0,0 +1,494 @@
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+    |        |        |        |        |        |        |      |        |        |        |        |        |       |
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+                                    ,-----------------------,  ,-----------------------.
+                                    |       |       |       |  |       |       |       |
+                                   /|-------+-------+-------'  +-------+-------+-------|\
+  ),
+*/
+  [_SODA] = LAYOUT_split_3x6_3(
+    _______,  _______,  KC_UP,  _______,  _______,  _______,        _______, _______, _______, _______, _______, _______,
+    _______,  KC_LEFT,  KC_DOWN,  KC_RGHT,  _______,  _______,      _______, _______, _______, _______, _______, _______,
+    _______,  _______,  _______,  _______,  _______,  _______,      _______, _______, _______, _______, _______, _______,
+                                      _______, _______, _______,    _______, _______, _______
+  ),
+
+
+
+/*
+ * ADJUST LAYER
+  [_TPL] = LAYOUT_wrapper( \
+  //,-----------------------------------------------------.      ,----------------------------------------------------.
+    |  RESET | EEPRST |        |        |        |        |      |        |        |        |        |        |       |
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+    | RGBTog |  HUE▲  |  SAT▲  | BRGHT▲ |        |        |      |        |        |        |        |        |       |
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+    | RGBMOD |  HUE▼  |  SAt▼  | BRGHT▼ |        |        |      |        |        |        |        |        |       |
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+                                    ,-----------------------,  ,-----------------------.
+                                    |   OS  |       | Space |  |  Ent  |       |  ALT  |
+                                   /|-------+-------+-------'  +-------+-------+-------|\
+  ),
+*/
+  [_ADJUST] = LAYOUT_split_3x6_3(
+    RESET, EEP_RST, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
+    RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, _______, _______,                    _______, _______, _______, _______, _______, _______,
+    RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, _______, _______,                    _______, _______, _______, _______, _______, _______,
+                                          KC_LGUI, _______,  KC_SPC,     KC_ENT, _______, KC_RALT
+  )
+};
+
+
+#ifdef OLED_DRIVER_ENABLE
```

Comment:
```suggestion
#ifdef OLED_ENABLE
```

Reply:
done in https://github.com/qmk/qmk_firmware/pull/13745/commits/cedae37b86761fbd6bbf7d4a689181e015e5f946

Reply:
Hey, this didn't appear to get merged to the main branch.

Reply:
Aha, nice catch @krikun98 , thanks! I've just created #16284 regarding this.

Reply:
@Ardakilic, would you happen to have a branch/hex with the working animation? Even after fixing this bug the WPM counter is just stuck at 0 on my board.

Reply:
Hey @krikun98 , sadly I sold my current Corne (classic) to one of my friends, but a low profile PCB is produced from factory and is on the way. I can't sadly test right now.

However, I took the animation from here: https://github.com/nwii/oledbongocat , it should work nicely.

Also, 

```c
char wpm_str[10];
void oled_task_user(void) {
sprintf(wpm_str, "WPM:%03d", get_current_wpm())
oled_write(wpm_str, false);
}
```

lines should suffice for wpm counter, regardless of the animation.

Also check this, it provides two types of examples for WPM: https://docs.qmk.fm/#/squeezing_avr?id=oled-tweaks

Reply:
The issue was the lack of `#define SPLIT_WPM_ENABLE` in the code, the central half has no way to communicate WPM to the peripheral otherwise. This should probably be included?